include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -707,12 +731,26 @@ appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
                      TScores& scores,
                      LocalPairwiseLibrary)
 {
+    appendSegmentMatches(str, pList, score_type, matches, scores, LocalPairwiseLibrary(), OrdinaryAlignment());
+}
+
+//////////////////////////////////////////////////////////////////////////////
+
+template<typename TString, typename TSpec, typename TSize2, typename TSpec2, typename TScore, typename TSegmentMatches, typename TScoreValues>
+inline void
+appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
+                     String<TSize2, TSpec2> const& pList,
+                     TScore const& score_type,
+                     TSegmentMatches& matches,
+                     TScoreValues& scores,
+                     LocalPairwiseLibrary,
+                     OrdinaryAlignment)
+{
```

Comment:
Yes, I remember about our parameters order conventions. But the problem is the old code had been written before these conventions were established. At least, it seems so. Look for example at other `appendSegmentMatches` functions (line 593 or 640 [here](https://github.com/oyasnev/seqan/blob/feature/deep_coffee/include/seqan/graph_msa/graph_align_tcoffee_library.h)) So I had to support the same style, otherwise we need to change **all** these functions' signatures, which might break backward compatibility. That was my point.